tests/test_iot.py

```diff
@@ -12,7 +12,6 @@
 
 @pytest.fixture(scope='module')
 def iot_settings():
-    settings.IOT_THING_INTEGRATION = True
```

Comment:
Any reason why this was here before and is not needed anymore? Or was it never needed?

Reply:
It was never needed for these unit tests. This was copied from either transmission or profile's setup for the tests, where we needed to ensure that the AWSClient was being reached.